Source/Rhetos.Compiler/Rhetos.Compiler.csproj

```diff
@@ -247,7 +133,17 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <None Include="packages.config" />
+    <PackageReference Include="Microsoft.CodeAnalysis.Analyzers">

```

Comment:
Microsoft.CodeAnalysis.Analyzers should probably be removed. It is just a dependency for Microsoft.CodeAnalysis.CSharp.